lib/functions/rootfs/distro-specific.sh

```diff
@@ -188,22 +190,44 @@ function create_sources_list() {
 		chroot "${basedir}" /bin/bash -c "cat armbian.key | apt-key add -"
 	fi
 
+	declare -a components=()
+	if [[ "${when}" == "image" ]]; then # only include the 'main' component when deploying to image
+		components+=("main")
+	fi
+	components+=("${RELEASE}-utils")   # utils contains packages Igor picks from other repos
```

Comment:
@igorpecovnik is this even true?

Reply:
Initially it was a place for things we built in extrapackages so its more generic, now it serves only for manually added packages from 3rd party repo.